ChildProject/pipelines/metrics.py

```diff
@@ -152,188 +410,53 @@ def __init__(
         self,
         project: ChildProject.projects.ChildProject,
         set: str,
```

Comment:
Why is `set` an argument for a specific type of metrics, like LENA here? Could there be multiple sets of LENA annotations? Or can the user choose a custom name for the set? Sorry, I am still a bit confused about the data structure and sets.

BTW, is there a glossary of sorts? With things like "recording", "set", "pipeline", etc.? Obviously, you guys all know that stuff but it might be helpful to newcomers.

Reply:
The use of 'its' as the name of the lena set is what we do by default but yes, this allows people to give the name they want to their lena set.
At the moment, we don't really have a glossary, that sounds like a really good idea to improve communication 👍 . We often ourselves get mixed up between words like "segments"/ "annotations" ...